integration-tests/realm-apps/with-db/auth_providers/local-userpass.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "local-userpass",
```

Comment:
Could this re-use https://github.com/realm/realm-js/tree/bindgen/integration-tests/realm-apps/with-email-password instead? I'd like for us to find a way to (eventually) move more of this configuration into the test file 🤔

Reply:
it could probably, right now the thing I've done is to put everything in the `with-db` App, but I'm pretty sure that it will become quite bloated once all tests are merged. But I'm trying to come up with an idea of how the apps can be a bit more modular...